backend/serializers/participant_serializer.py

```diff
@@ -4,10 +4,12 @@
 class ParticipantSchema(Schema):
     class Meta:
         fields = (
-            'id', 'name', 'lastname', 'email', 'github', 'hacknights', 'phone'
+            'id', 'participants_ids', 'name', 'lastname',
+            'email', 'github', 'hacknights', 'phone'
         )
         dump_only = ('id', 'hacknights')
 
+    participants_ids = fields.List(fields.Int())
```

Comment:
Wait, how the `participants_ids` as a list can be part of a `ParticipantSchema`? I can agree, that list in such shape will be used to attach participants to a hacknight, but it should be defined in probably different place.